app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsFragment.kt

```diff
@@ -80,6 +100,41 @@ class SettingsFragment : PreferenceFragment() {
         }
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (REQUEST_CODE_ALARM_TONE == requestCode && data != null) {
```

Comment:
This should probably also check for `resultCode == RESULT_OK`.

Reply:
:heavy_check_mark: Changed in 23b10ca4822a133ed18915842987e132b4a9205a.